### PR TITLE
Test delete of konk and konkservice resources

### PR DESCRIPTION
**Pull request by kd7lxl**

### CODE REVIEW

File: .github/workflows/pr.yaml [hunk 1]
```diff
@@ -123,6 +123,10 @@ jobs:
           do
             sleep 3
           done
+      - name: Test delete
+        timeout-minutes: 1
+        run: |
+          kubectl delete konks,konkservices --all --all-namespaces
```
Comment (drewwells) [inline .github/workflows/pr.yaml:129]: We should check with helm that the operator actually completed the task.
```suggestion
          kubectl delete konks,konkservices --all --all-namespaces
          helm -n $KONK_NAMESPACE | grep -c runner-konk 
```

Comment (drewwells) [reply .github/workflows/pr.yaml:129]: might need to add `set -e` to get github-actions to fail on matches

Comment (kd7lxl) [reply .github/workflows/pr.yaml:129]: `set -e` is default in actions.

Comment (kd7lxl) [reply .github/workflows/pr.yaml:131]: Added support for this.

**Review by drewwells — APPROVED** [review]

**Pull request merged**